### PR TITLE
added readme to complexity katas repo

### PR DESCRIPTION
**Pull request by randyriback**
This is a test edit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This is Randy's proposed PR
```
Comment (rigdern) [inline README.md]: Can you end the sentence with a period?

Comment (randyriback) [reply README.md]: Sure thing

**Comment by rigdern** [comment]
Looks pretty good. Can you address the 1 comment I left?

**Comment by rigdern** [comment]
Thanks for the PR. It looks good! Merging.

**Pull request merged**